### DISCUSSION TITLE
Was there an option for selected word highlighting?

### DISCUSSION THREAD

**Discussion by dados** [pyscripter/SynEdit#98] [General]
Was there an option for selected word highlighting?
So that all other same words would be highlighted

**Comment by pyscripter** [comment]
It is not built in.  But you can easily implement it using Indicators.
See HighligthtSearchTerm in https://github.com/pyscripter/SynEdit/blob/master/Demos/SearchReplaceDemo/uSearchHighlighter.pas.

You can highlight the selected word by:
-  post processing ecSelWord and
- handling the OnDblClick event
- then clearing the indicators when the caret or selection changes. (e.g. OnStatusChange).  

PyScripter does that.

**Reply by dados** [reply]
Thanks
